datayoga-py/datayoga/blocks/std/read/block.py

```diff
@@ -0,0 +1,42 @@
+import ast
+import logging
+import select
+import sys
+import uuid
+from typing import Any, Dict, Generator, List, Optional
+
+import aioconsole
+from datayoga.context import Context
+from datayoga.producer import Message
+from datayoga.producer import Producer as DyProducer
+
+logger = logging.getLogger("dy")
+
+
+class Block(DyProducer):
+
+    def init(self, context: Optional[Context] = None):
+        logger.debug(f"Initializing {self.get_block_name()}")
+
+    async def produce(self) -> Generator[Message, None, None]:
+        def get_records(data: str) -> List[Dict[str, Any]]:
+            records = ast.literal_eval(data)
```

Comment:
what is this for? the literal_eval always seems freightnin because of injection risk (although we went through it before and it is probably safe). I would use something explicit. Or remove completely.

Reply:
it takes a string and converts it into a dict without caring to much if ' or " is used (while json.loads shouts on one of them). What do you suggest to use instead?

Reply:
fix to use json.loads